{"body": "Oh, so it seems that it went through, I tried a lot of different git incantations and nothing was happening... ", "url": "https://api.github.com/repos/rails/rails/issues/comments/4845372", "created_at": "2012-03-30T15:58:43Z", "html_url": "https://github.com/rails/rails/pull/5328#issuecomment-4845372", "updated_at": "2012-03-30T15:58:43Z", "repo": "rails", "issue_id": 5328, "user": {"following_url": "https://api.github.com/users/gampleman/following{/other_user}", "events_url": "https://api.github.com/users/gampleman/events{/privacy}", "organizations_url": "https://api.github.com/users/gampleman/orgs", "url": "https://api.github.com/users/gampleman", "gists_url": "https://api.github.com/users/gampleman/gists{/gist_id}", "html_url": "https://github.com/gampleman", "subscriptions_url": "https://api.github.com/users/gampleman/subscriptions", "avatar_url": "https://2.gravatar.com/avatar/434d6ad623da43d14afbdefb9c0291c4?d=https%3A%2F%2Fidenticons.github.com%2Fd8b6528bbdf1c0f53259b92ada98d1c6.png", "repos_url": "https://api.github.com/users/gampleman/repos", "received_events_url": "https://api.github.com/users/gampleman/received_events", "gravatar_id": "434d6ad623da43d14afbdefb9c0291c4", "starred_url": "https://api.github.com/users/gampleman/starred{/owner}{/repo}", "login": "gampleman", "type": "User", "id": 69144, "followers_url": "https://api.github.com/users/gampleman/followers"}, "owner": "rails", "_id": {"$oid": "523819c6bd3543c15100e262"}, "id": 4845372, "issue_url": "https://api.github.com/repos/rails/rails/issues/5328"}
